mantle/cmd/kola/testiso.go

```diff
@@ -336,6 +336,9 @@ func runTestIso(cmd *cobra.Command, args []string) error {
 		if kola.CosaBuild.Meta.BuildArtifacts.LiveIso == nil {
 			return fmt.Errorf("build %s has no live ISO", kola.CosaBuild.Meta.Name)
 		}
+		if strings.Contains(kola.CosaBuild.Meta.BuildID, ".dev.") {
+			baseInst.Insecure = true //Dynamically setting test signing by default when running with development build
+		}
```

Comment:
Since the comment is long let's put it on a separate line. Also made
some wording change suggestions for the comment.

```suggestion
		if strings.Contains(kola.CosaBuild.Meta.BuildID, ".dev.") {
            // Default to ignore signing verification when running with development build
			baseInst.Insecure = true
		}
```